src/session.rs

```diff
@@ -374,13 +378,27 @@ impl Session {
             let stream_mid_rid = self.streams.stream_rx_by_mid_rid(mid, Some(rid));
 
             let ssrc_main = if is_repair {
-                stream_mid_rid?.ssrc()
+                let Some(stream) = stream_mid_rid else {
+                    return None;
+                };
+                let main = stream.ssrc();
+
+                if let Some(rtx) = stream.rtx() {
+                    // We got a change in the RTX SSRC
```

Comment:
Is this comment right?

Reply:
The comment is right, but the condition isn't